selene/core/condition.py

```diff
@@ -44,7 +44,7 @@ def fn(entity):
     @classmethod
     def by_or(cls, *conditions):
         def fn(entity):
-            errors: List[Exception] = []
+            errors: list[Exception] = []
```

Comment:
It's not python 3.7 - it's python 3.9 feature https://www.python.org/dev/peps/pep-0585/
It shouldn't be here for now.